frontend/appflowy_tauri/src/appflowy_app/stores/effects/folder/app/app_observer.ts

```diff
@@ -1,19 +1,20 @@
-import { Ok, Result } from "ts-results";
-import { FlowyError, FolderNotification, RepeatedViewPB } from "@/services/backend";
-import { ChangeNotifier } from "$app/utils/change_notifier";
-import { FolderNotificationObserver } from "../notifications/observer";
+import { Ok, Result } from 'ts-results';
+import { FlowyError, FolderNotification, RepeatedViewPB } from '@/services/backend';
+import { ChangeNotifier } from '$app/utils/change_notifier';
+import { FolderNotificationObserver } from '../notifications/observer';
 
 export type AppUpdateNotifyCallback = (value: Result<RepeatedViewPB, FlowyError>) => void;
 
 export class AppObserver {
   _appNotifier = new ChangeNotifier<Result<RepeatedViewPB, FlowyError>>();
+  _viewsNotifier = new ChangeNotifier<void>();
   _listener?: FolderNotificationObserver;
 
-  constructor(public readonly appId: string) {
-  }
+  constructor(public readonly appId: string) {}
 
-  subscribe = async (callbacks: { onAppChanged: AppUpdateNotifyCallback }) => {
+  subscribe = async (callbacks: { onAppChanged: AppUpdateNotifyCallback; onViewsChanged: () => void }) => {
     this._appNotifier?.observer?.subscribe(callbacks.onAppChanged);
+    this._viewsNotifier?.observer?.subscribe(callbacks.onViewsChanged);
```

Comment:
_viewsNotifier should be unsubscribe when the observer is unsubscribe, I forgot before. Could you add it?

Reply:
done